docs/UserGuide.adoc

```diff
@@ -60,10 +79,24 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 Format: `help`
 
-=== Adding a person: `add`
+=== Log in to LinkedIn : 'linkedin_login' (Half implemented)
```

Comment:
Do not add non-functioning functions for the user to use, as every version increase should be a fully working product. Keep in-progress code in the personal repo and try not to merge it into the team repo